doc/Core.xml

```diff
@@ -7610,8 +7610,8 @@ http://www.onvif.org/ver10/tev/topicExpression/ConcreteSet
               <para>
                 <emphasis role="bold">PublishFilter</emphasis>
               </para>
-              <para>Concrete Topic Expression to select specific topics to publish, see section <xref linkend="_Ref210199558" />.</para>
-			  <para>If PublishFilter is empty, then device shall send all events but not metadata, for e.g. "MyDevice/onvif-ej//." <xref linkend="_Ref210199558" />.</para>
+              <para>Concrete Topic Expression to select specific topics to publish, see section <xref linkend="_Ref210199558" />. Example TopicExpression: "tns1:VideoAnalytics//.|tns1:RuleEngine//."</para>
+			        <para>If PublishFilter is empty, then device shall send all events but not metadata, for e.g. "MyDevice/onvif-ej//." <xref linkend="_Ref210199558" />.</para>
```

Comment:
Publish filter empty case is clear, but its not clear how we can set the publish filter for event and metadata topic. Here in example, its following the eventservice topic filter concept, do we have to use the full base topic "mydevice/onvif-ej/VideoAnalytics//.|mydevice/onvif-mj/VideoAnalytics" ?

Reply:
Your example is what actually also some implementer tried for normal events and complained that it didn't work. I don't think that it is a good idea to mix MQTT topic expression with ONVIF topic expression.

Should we keep this as open issue or do we want to fix it before release?

Reply:
Generally agree. However MetadataPublishFilter empty sending everything seems to be wrong, because than it is always on.

Question: what is the filter syntax?

Reply:
Regarding the filter syntax, I can include an additional Metadata type as “Events” along with the “VideoAnalytics/AudioAnalytics/PTZ” and have a filter as below

MyDevice/onvif-mj/**Events**/RuleEngine/LineDetector/Crossed&amp;/1/2/MyImportantFence1
MyDevice/onvif-mj/VideoAnalytics/1/MyClassifier

Reply:
MetadataPublishFilter empty sending everything seems to be wrong, because than it is always on.
> But the same can be switched off with MetadataConfiguration as that's basic for all these filters to apply, Isn't it?

Reply:
Think I got the all/nothing mechanism: When MetadataFilter is missing no metadata is sent and when it is present with empty string then everything is sent.

Only part missing is to describe the format which probably deviates from PublishFilter.